src/test/java/net/sf/jabref/logic/layout/format/RTFCharsTest.java

```diff
@@ -49,51 +47,123 @@ public void testLaTeXHighlighting() {
     }
 
     @Test
-    @Ignore
     public void testComplicated() {
-        Assert.assertEquals("R\\u233eflexions sur le timing de la quantit\\u233e \\u230ae should be \\u230ae",
-                formatter.format("Réflexions sur le timing de la quantité \\ae should be æ"));
-
-        Assert.assertEquals("h\\u225all{\\uc2\\u339oe}", formatter.format("h\\'all\\oe "));
+        Assert.assertEquals("R\\u233eflexions sur le timing de la quantit\\u233e {\\u230ae} should be \\u230ae",
+                formatter.format("Réflexions sur le timing de la quantité {\\ae} should be æ"));
+
+        Assert.assertEquals("h\\'e1ll{\\u339oe}", formatter.format("h\\'all{\\oe}"));
```

Comment:
Some test methods inside this test class have lots of assertions for single characters, can I leave it like that? I will split the longer ones like you recommend.